Makefile

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
This mostly comes from the streak.club repo. And overtime make we can move a couple things out of bin/ to this.